test/e2e/framework/util.go

```diff
@@ -2803,16 +2803,22 @@ func getReplicasFromRuntimeObject(obj runtime.Object) (int32, error) {
 			return *typed.Spec.Replicas, nil
 		}
 		return 0, nil
-	case *extensionsinternal.ReplicaSet:
-		return typed.Spec.Replicas, nil
-	case *extensionsinternal.Deployment:
-		return typed.Spec.Replicas, nil
+	case *extensions.ReplicaSet:
+		if typed.Spec.Replicas != nil {
+			return *typed.Spec.Replicas, nil
+		}
+		return 0, nil
+	case *extensions.Deployment:
+		if typed.Spec.Replicas != nil {
+			return *typed.Spec.Replicas, nil
+		}
+		return 0, nil
```

Comment:
The first two commits are touching the same code - can you please squash those two?